doc/installation/system/inifile.rst

```diff
@@ -112,6 +112,14 @@ If ``PI_HSM_PADDING`` is not set, the default internal padding is used::
 .. warning:: The two paddings are not compatible. Data encrypted with one padding can not decrypted with the other padding. 
    So if you change the padding later during the cause of operation, it will not be possible to decrypt old data!
 
+``PI_PREFERRED_LANGUAGE`` is a list in which the preferred languages can be defined. The browser's language settings are compared to this
+list and the "best match" wins. If none of the languages set in the browser match, the first language in the list will be used as the default language::
+
+    PI_PREFERRED_LANGUAGE = ["en", "de", "es", "fr"]
+
+.. note:: If ``PI_PREFERRED_LANGUAGE`` is not defined, a ``DEFAULT_LANGUAGE_LIST`` is used.
```

Comment:
just remove the `DEFAULT_LANGUAGE_LIST` and replace it with something like "... the following list is used"

Reply:
fixed in be6d9a01cd35589755d7885117001b48ceaca91c.